athlinks-ontology.ttl

```diff
@@ -6,542 +6,692 @@
-           dc:creator "nextdude"^^xsd:string ;
-           dc:date "2017-05-22T05:16:50Z"^^xsd:dateTime ;
-           rdfs:comment "A team of athletes in an athletic event that each run part of the event course and their overall time is the sum of their individual times."@en ;
-           rdfs:label "Relay team"@en .
-
-
-###  http://adp.athlinks.com/schema/RunAction
-:RunAction rdf:type owl:Class ;
-           rdfs:subClassOf schema:ExerciseAction ;
-           dc:creator "nextdude"^^xsd:string ;
-           dc:date "2017-07-26T14:06:45Z"^^xsd:dateTime ;
-           rdfs:comment "Exercise by running"@en ;
-           rdfs:label "Run Action"@en .
-
-
-###  http://adp.athlinks.com/schema/SwimAction
-:SwimAction rdf:type owl:Class ;
-            rdfs:subClassOf schema:ExerciseAction ;
-            dc:creator "nextdude"^^xsd:string ;
-            dc:date "2017-07-26T14:06:57Z"^^xsd:dateTime ;
-            rdfs:comment "Exercise by swimming"@en ;
-            rdfs:label "Swim Action"@en .
-
-###  http://adp.athlinks.com/schema/CohortRankingType
-:CohortRankingType rdf:type owl:Class ;
-  rdfs:subClassOf schema:Intangible ;
-  rdfs:comment "A predefined rule for associating entries to a cohort."@en ;
-  rdfs:label "Cohort Ranking Type"@en .
-
-:cohortRankingType a owl:ObjectProperty ;
-  rdfs:domain :CohortRanking ;
-  rdfs:range [
-    rdf:type owl:Class ;
-    owl:unionOf(
-      :CohortRankingType
-      xsd:string
-    )
-  ] ;
-  rdfs:comment "The cohort ranking type of this CohortRanking; either one of the predefined ones, or a string denoting a custom cohort type (e.g. military)."@en ;
-  rdfs:label "cohort ranking type"@en .
-
-###  http://adp.athlinks.com/schema/OverallRanking
-:OverallRanking a owl:NamedIndividual, :CohortRankingType ;
-  rdfs:label "Overall Ranking"@en .
-
-:GenderRankingMale a owl:NamedIndividual, :CohortRankingType ;
-  rdfs:comment "Cohort Ranking Type for male athletes."@en ;
-  rdfs:label "Gender Ranking: Male"@en .
-
-:GenderRankingFemale a owl:NamedIndividual, :CohortRankingType ;
-  rdfs:comment "Cohort Ranking Type for female athletes."@en ;
-  rdfs:label "Gender Ranking: Female"@en .
-
-:GenderRankingUnspecified a owl:NamedIndividual, :CohortRankingType ;
-  rdfs:comment "Cohort Ranking Type for athletes with unspecified gender"@en ;
-  rdfs:label "Gender Ranking: Unspecified"@en .
-
-###  http://adp.athlinks.com/schema/CohortRanking
-:CohortRanking rdf:type owl:Class ;
-  rdfs:subClassOf schema:Intangible ;
-  rdfs:comment "An individual result's ranking in a cohort."@en ;
-  rdfs:label "Cohort Ranking" .
-  # Expected properties: 
-  #  :athleteCohort -> :AthleteCohort
-  #  :rank -> Int
-
-:cohortRanking a owl:ObjectProperty ;
-  rdfs:domain :AthleteResult ;
-  rdfs:range :CohortRanking ;
-  rdfs:comment "A ranking of the individual result within a cohort."@en ;
-  rdfs:label "cohort ranking"@en .
-
-###  http://adp.athlinks.com/schema/rank
+	rdfs:domain schema:Person ;
```

Comment:
This begs the question as to whether this is a transactional or warehouse system. Building flink jobs to handle updating this data might become a bit of a burden at some point when in reality we can just use the graph nature of the database to do these calculations, right?  Or is this being put in because we can't translate the graph nature of our database to graphql (graphql doesn't understand the concept of counting the number of links between items, so we'd just have to return the items and count them in the code)?

Reply:
I don't think we're using this property at all in the first place, now would be a great time to delete it.

Reply:
Sorry, this comment was supposed to be on numberOfRaces

Reply:
I would love to delete numberOfRaces.

Reply:
Even better!

Reply:
It's gone in the most recent change.